packages/mural-card/src/components/mural-card/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { Card, CardActionArea, CardMedia } from '@material-ui/core';
+import { EventHandler } from '@muraldevkit/mural-integrations-common';
+import * as React from 'react';
+import styled from 'styled-components';
+
+export type CardSize = 'normal' | 'small';
+
+export type CardItemSource = {
+  title: string;
+  thumbnailUrl: URL | string;
+  details?: string;
+  initials?: string;
+};
+
+export interface MuralCardPropTypes {
+  source: CardItemSource;
+  onClick: EventHandler;
```

Comment:
Maybe this `onClick` could be optional? I can think of a use case in Teams where we would like to use this component without a click event.